### PR TITLE
Feature: single graph toggle 

### PR DESCRIPTION
**Pull request by shmuel-web**
### TL;DR
new feature 
+ single graph comparison
+ clear label that makes it easy to distinguish the fastest provider
+ toggle to switch between views

### Description:
Hi Bernd

as discussed I added a new view that doesn't overload the existing view, 
but still provides a single graph comparison of all providers.
so a single graph comparison
with a clear label that makes it easy to distinguish the fastest provider
and a toggle to switch between the single graph to the individual provider graphs which are more esthetic but does not tell us much as they are on different scale :)
the toggle is off by default.

### snapshots:
Screen Shot 2019-12-23 at 14 29 30
Screen Shot 2019-12-23 at 14 30 20
Screen Shot 2019-12-23 at 14 30 33

### CODE REVIEW

**Comment by Strernd** [comment]
Thanks very much, I like this feature! Great work :)
